src/localisation/lang/nl/locale.json

```diff
@@ -41,6 +41,7 @@
     "more-info": "Meer informatie",
     "track": "Track",
     "playlists": "Playlists",
+    "timer": "Set Sleep Timer",
```

Comment:
Only add this if you're adding the translated string. If not, remove it, so that translation can be picked up after the PR is merged.